mavlink/Cargo.toml

```diff
@@ -19,15 +19,15 @@ edition = "2018"
 rust-version = "1.65.0"
 
 [build-dependencies]
-mavlink-bindgen = { version = "0.13.1", default-features = false }
+mavlink-bindgen = { path = "../mavlink-bindgen", version = "0.13.2", default-features = false }
 
 [[example]]
 name = "mavlink-dump"
 path = "examples/mavlink-dump/src/main.rs"
 required-features = ["ardupilotmega"]
 
 [dependencies]
-mavlink-core = { version = "0.13.1", default-features = false }
```

Comment:
You should avoid the following for the release system to work.

Reply:
I have removed the versions in 33b4d4ec3c971bb1af81080c31a77304a790096e but this breaks most tests.
Looking at the failed build log of 9cd7bf0b8c720d255232a389051fb60f5fc92c2c before the path dependencies where removed the issue seems to be that no version at all is specified:
> error: all dependencies must have a version specified when publishing.

But since cargo supports specifying a [path and a version](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#multiple-locations) doing so should circumvent the issue.

Reply:
resolved based with merged PR #255 that revert to using paths